Userland/Applications/TerminalSettings/CMakeLists.txt

```diff
@@ -4,18 +4,16 @@ serenity_component(
     TARGETS TerminalSettings
 )
 
-stringify_gml(TerminalSettingsMain.gml TerminalSettingsMainGML.h terminal_settings_main_gml)
-stringify_gml(TerminalSettingsView.gml TerminalSettingsViewGML.h terminal_settings_view_gml)
+compile_gml(TerminalSettingsMain.gml TerminalSettingsMainGML.cpp)
+compile_gml(TerminalSettingsView.gml TerminalSettingsViewGML.cpp)
 
 set(SOURCES
-    TerminalSettingsWidget.cpp
+    TerminalSettingsViewGML.cpp
+    TerminalSettingsMainGML.cpp
+    MainWidget.cpp
+    ViewWidget.cpp
```

Comment:
This is a very big commit. Splitting the file into `MainWidget` and `ViewWidget` should probably be done beforehand (in a separate commit) or not at all.